doc/user-guides/full-walkthrough/secure-protect-connect-openshift.md

```diff
@@ -107,418 +140,261 @@ spec:
-
-### Step 4a - (Optional) Configure metrics to be scraped from the Gateway instance
-
-If you have prometheus in your cluster, set up a PodMonitor to configure it to scrape metrics directly from the Gateway pod.
-This must be done in the namespace where the Gateway is running.
-This configuration is required for metrics such as `istio_requests_total`.
+Check the status of the listener, you will see that it is not yet programmed or ready to accept traffic due to bad TLS configuration. This will be fixed in the next step with the `TLSPolicy`:
 
 ```bash
-kubectl apply -f - <<EOF
-apiVersion: monitoring.coreos.com/v1
-kind: PodMonitor
-metadata:
-  name: istio-proxies-monitor
-  namespace: ${gatewayNS}
-spec:
-  selector:
-    matchExpressions:
-      - key: istio-prometheus-ignore
-        operator: DoesNotExist
-  podMetricsEndpoints:
-    - path: /stats/prometheus
-      interval: 30s
-      relabelings:
-        - action: keep
-          sourceLabels: ["__meta_kubernetes_pod_container_name"]
-          regex: "istio-proxy"
-        - action: keep
-          sourceLabels:
-            ["__meta_kubernetes_pod_annotationpresent_prometheus_io_scrape"]
-        - action: replace
-          regex: (\d+);(([A-Fa-f0-9]{1,4}::?){1,7}[A-Fa-f0-9]{1,4})
-          replacement: "[\$2]:\$1"
-          sourceLabels:
-            [
-              "__meta_kubernetes_pod_annotation_prometheus_io_port",
-              "__meta_kubernetes_pod_ip",
-            ]
-          targetLabel: "__address__"
-        - action: replace
-          regex: (\d+);((([0-9]+?)(\.|$)){4})
-          replacement: "\$2:\$1"
-          sourceLabels:
-            [
-              "__meta_kubernetes_pod_annotation_prometheus_io_port",
-              "__meta_kubernetes_pod_ip",
-            ]
-          targetLabel: "__address__"
-        - action: labeldrop
-          regex: "__meta_kubernetes_pod_label_(.+)"
-        - sourceLabels: ["__meta_kubernetes_namespace"]
-          action: replace
-          targetLabel: namespace
-        - sourceLabels: ["__meta_kubernetes_pod_name"]
-          action: replace
-          targetLabel: pod_name
-EOF
+kubectl get gateway ${KUADRANT_GATEWAY_NAME} -n ${KUADRANT_GATEWAY_NS} -o=jsonpath='{.status.listeners[0].conditions[?(@.type=="Programmed")].message}'
 ```
 
-### Step 5 - Secure and protect the Gateway with auth, TLS, rate limit, and DNS policies
-
-While your Gateway is now deployed, it has no exposed endpoints and your listener is not programmed. Next, you can set up a `TLSPolicy` that leverages your CertificateIssuer to set up your listener certificates. 
-
-You will also define an `AuthPolicy` that will set up a default `403` response for any unprotected endpoints, as well as a `RateLimitPolicy` that will set up a default artificially low global limit to further protect any endpoints exposed by this Gateway.
-
-#### Set the Auth policy
-
-Set a default, deny-all `AuthPolicy` for your Gateway as follows:
+### Deploy the gateway TLS policy
```

Comment:
We are securing the gateway in this step also as TLS based communication is a key piece to secure communications between client and gateway

Reply:
ok ill move the secure the gateway heading up further